kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/BaseEntityService.java

```diff
@@ -312,4 +312,32 @@ public void setupBaseEntity(String baseEntityCode, String... attributeCodeValue)
 		beUtils.updateBaseEntity(be);
 	}
 
+	/**
+	 * Initialize the base entity and assign actual definition
+	 * @param definitionCode Base definition code
+	 * @param detailDefCode Detail definition code
+	 * @return
+	 */
+	public String init(String definitionCode,String detailDefCode) {
```

Comment:
Please elaborate on the purpose of this method

Reply:
Add more comments in the method